File: nas-music/src/main/java/top/itning/yunshunas/music/entity/Music.java
package top.itning.yunshunas.music.entity;

import lombok.Data;

import java.util.Date;

/**
 * @author itning
 * @since 2020/9/5 11:08
 */
@Data
public class Music {
    /**
     * 数据库主键ID
     */
    private long id;
    /**
     * 音乐ID
     */
    private String musicId;
    /**
     * 音乐名
     */
    private String name;
    /**
     * 歌手
     */
    private String singer;
    /**
     * 歌词ID
     */
    private String lyricId;
    /**
     * 音乐类型
     * {@link top.itning.yunshunas.music.constant.MusicType}
     */
    private Integer type;
    /**
     * 创建时间
     */
    private Date gmtCreate;
    /**
     * 更新时间
     */
    private Date gmtModified;

}

File: nas-front-end/src/app/http/model/Music.ts
export class Music {
  /**
   * 音乐ID
   */
  musicId: string;
  /**
   * 音乐名
   */
  name: string;
  /**
   * 歌手
   */
  singer: string;
  /**
   * 歌词ID
   */
  lyricId: string;
  /**
   * 音乐类型
   */
  type: number;

  /**
   * 音乐URL
   */
  musicUri: string;

  /**
   * 歌词URL
   */
  lyricUri: string;

  /**
   * 封面URL
   */
  coverUri: string;

  /**
   * 创建时间
   */
  gmtCreate: string;

  /**
   * 修改时间
   */
  gmtModified: string;
}
